### DISCUSSION TITLE
Calendar => [HOW TO] Disable automatic month switch.

### DISCUSSION THREAD

**Discussion by ehcaneontheweb** [rsuite/rsuite#4065] [Q&A]
When clicking on a day outside of the current month, for example 1st of next month, the calendar automatically switches to next month. Is there a way do disable that behaviour?

![Screenshot 2024-12-14 at 4 07 07 AM](https://github.com/user-attachments/assets/07ed57ae-67d8-4655-b581-2c5242e75c8d)

**Comment by SevenOutman** [comment]
Hey @ehcaneontheweb, what's your expected behavior when Next 1st is clicked?

**Comment by ehcaneontheweb** [comment]
To just select the date, but do not switch the month page visually. @SevenOutman
